public/app/features/alerting/unified/components/rule-viewer/RuleViewer.tsx

```diff
@@ -147,12 +147,13 @@ const createMetadata = (rule: CombinedRule): PageInfoItem[] => {
   const hasLabels = !isEmpty(labels);
 
   const interval = group.interval;
+  const styles = useStyles2(getStyles);
 
   if (runbookUrl) {
     /* TODO instead of truncating the string, we should use flex and text overflow properly to allow it to take up all of the horizontal space available */
     const truncatedUrl = truncate(runbookUrl, { length: 42 });
```

Comment:
I'm actually not sure why this doesn't shorten the string, tested in 11.3.0 which should have this

Reply:
I was going to remove it but the Text element doesn't have className to apply the style.